Master/index.js

```diff
@@ -13,7 +13,7 @@ const async = require('async');
 const parse = require('parse-key-value');
 
 const pkg = require('../package.json');
-const AlAzureMaster = require('@alertlogic/al-azure-collector-js').AlAzureMaster;
+const EhubCollectorMaster = require('./ehub_master').EhubCollectorMaster;
 const healthcheck = require('./healthcheck');
```

Comment:
pkg , healthcheck and APP_FUNCTIONS are moved in ehub_master.js,so we can remove from here.